generators/server/templates/src/test/resources/config/application.yml.ejs

```diff
@@ -51,7 +51,7 @@ eureka:
 
 <%_ if (databaseTypeMongodb) { _%>
 mongock:
-  migration-scan-package:
+  change-logs-scan-package:
```

Comment:
I'm not sure if this is even used during testing. `mvn verify` passes both with `migration-scan-package` and the new value.